operations/rack_resiliency/manage_critical_services.md

```diff
@@ -0,0 +1,394 @@
+# Editing/Deleting criticalservices from static configmap (rrs-mon-static)
+
+## Overview
+The rrs-mon-static ConfigMap in the rack-resiliency namespace contains configurations for critical services.
+
+Procedure to delete a criticalservice from the configmap
+Verify the ConfigMap exists:: kubectl get configmaps -n rack-resiliency
+
+### Example
+
+```bash
+kubectl get configmaps -n rack-resiliency 
+NAME                 DATA   AGE
+istio-ca-root-cert   1      6d21h
+kube-root-ca.crt     1      6d21h
+rrs-mon-dynamic      2      3d4h
+rrs-mon-static       11     3d4h
+```
+
+Our list of criticalservices is present in rrs-mon-static.
+To verify if the list is present run: kubectl describe configmap rrs-mon-static -n rack-resiliency
+
+### Example
+
+```bash
+kubectl describe configmap rrs-mon-static -n rack-resiliency 
+Name:         rrs-mon-static
+Namespace:    rack-resiliency
+Labels:       <none>
+Annotations:  <none>
+```
+
+### Data
+
+#### ceph_pre_monitoring_delay:
+60
+
+#### critical-service-config.json
+
+```json
```

Comment:
Having the `json` label here seems to be making the "..." get flagged as invalid syntax.   Same goes in other spots where you have "..." for brevity.   Maybe consider removing the "json"?